invenio_oaiserver/utils.py

```diff
@@ -79,3 +105,63 @@ def datetime_to_datestamp(dt, day_granularity=False):
     if day_granularity:
         result = result[:-10]
     return result
+
+
+def eprints_description(metadataPolicy, dataPolicy, content=None,
```

Comment:
Document parameters with the docstring could be useful since the docstring under configuration does not cover all parameters with examples. You can also add URL to each .xsd definition under each function for reference to the original definitions:

https://www.openarchives.org/OAI/2.0/oai-identifier.xsd
https://www.openarchives.org/OAI/2.0/eprints.xsd
https://www.openarchives.org/OAI/2.0/friends.xsd

Reply:
Good idea but this started being too verbose so I went with a description of the element and linked the full spec.